<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignupForm</class>
 <widget class="QWidget" name="SignupForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>785</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sign Up - Study with Bunny</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #E1F5FE;  /* M√†u n·ªÅn xanh pastel */</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_title">
     <property name="font">
      <font>
       <family>Chalkboard SE</family>
       <pointsize>36</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #FFF9C4;
    border-radius: 30px;</string>
     </property>
     <property name="text">
      <string>Sign Up for Study with Bunny üê∞</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="line_username">
     <property name="font">
      <font>
       <family>Chalkboard SE</family>
       <pointsize>23</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
    border: 2px solid #F88379;  /* vi·ªÅn h·ªìng ƒë√†o */
    border-radius: 8px;</string>
     </property>
     <property name="placeholderText">
      <string>T√™n ng∆∞·ªùi d√πng</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="line_email">
     <property name="font">
      <font>
       <family>Chalkboard SE</family>
       <pointsize>23</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
    border: 2px solid #F88379;  /* vi·ªÅn h·ªìng ƒë√†o */
    border-radius: 8px;</string>
     </property>
     <property name="placeholderText">
      <string>Email</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="line_password">
     <property name="font">
      <font>
       <family>Chalkboard SE</family>
       <pointsize>23</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
    border: 2px solid #F88379;  /* vi·ªÅn h·ªìng ƒë√†o */
    border-radius: 8px;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>M·∫≠t kh·∫©u</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="line_confirm">
     <property name="font">
      <font>
       <family>Chalkboard SE</family>
       <pointsize>23</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
    border: 2px solid #F88379;  /* vi·ªÅn h·ªìng ƒë√†o */
    border-radius: 8px;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Nh·∫≠p l·∫°i m·∫≠t kh·∫©u</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="rememberMeCheckBox">
     <property name="font">
      <font>
       <family>Chalkboard SE</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Remember me</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_signup">
     <property name="font">
      <font>
       <family>Chalkboard SE</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #FFD1DC;     /* N·ªÅn h·ªìng ƒë√†o */
    border: 2px solid #F88379;     /* Vi·ªÅn h·ªìng cam ƒë·∫≠m h∆°n */
    border-radius: 12px;</string>
     </property>
     <property name="text">
      <string>ƒêƒÉng k√Ω</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_login">
     <property name="font">
      <font>
       <family>Chalkboard SE</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px dashed #FFB6B9; /* vi·ªÅn ki·ªÉu ƒë·ª©t ƒëo·∫°n */
    border-radius: 12px;
    padding: 5px;</string>
     </property>
     <property name="text">
      <string>ƒêƒÉng nh·∫≠p ‚Üí</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="bunnyImageLabel">
     <property name="pixmap">
      <pixmap>../image/SPCK/·∫¢nh m√†n h√¨nh 2025-05-18 l√∫c 11.29.49.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>